Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -194,6 +192,8 @@ - (void)collectSignalWithParameters:(id<MASignalCollectionParameters>)parameters
     
     NSString *signal = [VungleAds getBiddingToken];
     [delegate didCollectSignal: signal];
+    
+    // TODO: We might need to send adaptive banner's width and height as extra param here.
```

Comment:
We cant do this at this point. we dont know if the placement is banner or fullscreen.

Reply:
They pass `parameters` which has all info we need, so we can covert adSize to VungleAdSize for in_line placement case. 

But, it might not necessary now, since our server can support "0" cases.